osp/models/form.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
+
+from osp.models.question import Question
+from osp.models.abstract_timestamp import AbstractTimestamp
+
+ALL = 'all'
+ADM = 'admin'
+STU = 'student'
+TARGET_USER = (
+    (ALL, 'All Users'),
+    (ADM, 'Admin'),
+    (STU, 'Student'),
+)
+
+class Form(AbstractTimestamp):
+
+    name = models.CharField(
+        max_length=255
+    )
+    description = models.TextField(
+        blank=True
+    )
+    created_on = models.DateTimeField(
```

Comment:
You are already inheriting `AbstractTimestamp`. no need to add these 2 fields again